### PR TITLE
Add MD3 component stickersheet to design resources

### PR DESCRIPTION
**Pull request by huayunh**
#### Changes proposed in this Pull Request:

- Added a link to the new Material Design 3 component sticker sheet
- Codeowner, replace Tom with Suraj

#### Screenshots / Screen Recording (if applicable)
Screenshot 2023-10-09 at 4 40 44 PM

#### To Test:

- yarn start
- http://localhost:3000/resources/designer

### CODE REVIEW

**Comment by huayunh** [comment]
Why did firebase deploy pass during the first commit but failed the following two?

File: src/__configuration__/design/resources.tsx [hunk 1]
```diff
@@ -24,9 +24,16 @@ export const patternLinks = [
 
 export const figmaCommunityFileLinks = [
     {
-        title: 'Brightlayer UI Component Sticker Sheet',
+        title: 'Brightlayer UI Components + Tokens',
+        url: '    https://www.figma.com/community/file/1293308181354933598/brightlayer-ui-tokens-components-in-material-design-3',
```
Comment (ektaghag-eaton) [inline src/__configuration__/design/resources.tsx]: Extra Space.

**Review by ektaghag-eaton — APPROVED** [review]
Changes are working fine 👍.

Screenshot 2023-10-10 at 11 32 40 AM

File: src/__configuration__/design/resources.tsx (hunk 1 above)
Comment (ektaghag-eaton) [reply src/__configuration__/design/resources.tsx]: Removed Extra space.

**Review by surajeaton — COMMENTED** [review]
Shall we point home page Figma Sticker Sheet to material design 3 style?
Currently, it is point to material design 2 style.

Screenshot 2023-10-10 at 12 19 57 PM

**Comment by huayunh** [comment]
>Shall we point home page Figma Sticker Sheet to material design 3 style?

I think we are not there yet, consider the rest of docit is still in Material Design 2

**Review by JeffGreiner-eaton — APPROVED** [review]

**Pull request merged**